app/src/main/java/com/github/h3lp3rs/h3lp/storage/LocalStorage.kt

```diff
@@ -22,15 +23,14 @@ class LocalStorage(private val path: String, context: Context, private val enabl
      * @throws NullPointerException if the user is not authenticated AND online sync is enabled.
      */
     fun pull(){
-        /*
-        if (enableOnlineSync) { // TODO: Need mocked version!
+        if (enableOnlineSync) {
             // Need to be authenticated if online sync is enabled
```

Comment:
Is there a way for the user to choose if he want the synch?

If no it is a good idea to add it to user stories

Reply:
Yes, you can add this in the enumeration! You cannot forge your own storage instance dynamically if that is your question. And I don't see where this could be useful. (cf. Storages enumeration)
If your question is about choosing whether data is ever put to the database (for privacy or backup constraints) this could indeed be a story for later! Let's add it!